workshop/exercises/pygeoapi.config.yml

```diff
@@ -184,7 +184,44 @@ resources:
               data: tests/data/ne_110m_admin_0_countries.sqlite
               id_field: ogc_fid
               table: ne_110m_admin_0_countries
-
+    sediments_sta:
+        type: collection
+        title: DDT 24' in Sédiments on Eau brute at L'AUXENCE A VIMPELLES 2 with method Méthode inconnue
+        description: DDT 24' in Sédiments on Eau brute at L'AUXENCE A VIMPELLES 2 with method Méthode inconnue
+        keywords:
+            - sediments
+        extents:
+            spatial:
+                bbox: [-180,-90,180,90]
+                crs: http://www.opengis.net/def/crs/OGC/1.3/CRS84
+        providers:
+            - type: feature
+              name: SensorThings
+              data: https://sensorthings-wq.brgm-rec.fr/FROST-Server/v1.0/
+              uri_field: uri
+              entity: Datastreams
+              time_field: phenomenonTime
+              intralink: true
```

Comment:
Please leave a blank line between this and the previous collection (as in the other examples)